build.sh

```diff
@@ -228,6 +235,34 @@ fi
 
 if [[ $_RET != 0 ]]; then
     echo "See error details above. Exit code was $_RET"
+else
+    if [[ $CREATE_DEB != 0 ]]; then
+        DEB_FOLDER=`realpath .`
+        DEB_FOLDER="${DEB_FOLDER}/chakracore_${CREATE_DEB}"
+
+        mkdir -p $DEB_FOLDER/usr/local/bin
+        mkdir -p $DEB_FOLDER/DEBIAN
+        cp $DEB_FOLDER/../ch $DEB_FOLDER/usr/local/bin/
+        if [[ $STATIC_LIBRARY == "" ]]; then
+            cp $DEB_FOLDER/../*.so $DEB_FOLDER/usr/local/bin/
+        fi
+        echo -e "Package: ChakraCore"\
+            "\nVersion: ${CREATE_DEB}"\
+            "\nSection: base"\
+            "\nPriority: optional"\
+            "\nArchitecture: amd64"\
+            "\nDepends: libc6 (>= 2.19), uuid-dev (>> 0), libunwind-dev (>> 0), libicu-dev (>> 0)"\
+            "\nMaintainer: ChakraCore <chakracore@microsoft.com>"\
+            "\nDescription: Chakra Core"\
+            "\n Open source Core of Chakra Javascript Engine"\
```

Comment:
@liminzhu @EdMaurer @bterlson Does this metadata look alright for a public release?